scripts/generate-helm.sh

```diff
@@ -278,22 +278,6 @@ if [ -f "${HELM_DIR}/templates/deployment.yaml" ]; then
       continue
     fi
     
-    # check if this is the problematic pod securityContext line
```

Comment:
should be fine to remove, i'm curious why this is always set to patch it like this